CONTRIBUTING.rst

```diff
@@ -0,0 +1,153 @@
+How to contribute to Flask-SQLAlchemy
+=====================================
+
+Thank you for considering contributing to Flask-SQLAlchemy!
+
+Support questions
+-----------------
+
+Please, don't use the issue tracker for this. Use one of the following
+resources for questions about your own code:
+
+* The IRC channel ``#pocoo`` on FreeNode.
+* The IRC channel ``#python`` on FreeNode for more general questions.
+* The mailing list flask@python.org for long term discussion or larger issues.
+* Ask on `Stack Overflow`_. Search with Google first using:
+  ``site:stackoverflow.com flask-sqlalchemy {search term, exception message, etc.}``
+
+.. _Stack Overflow: https://stackoverflow.com/questions/tagged/flask-sqlalchemy?sort=linked
+
+Reporting issues
```

Comment:
See also #728.  Do we want to duplicate content or just reference the issue template from here?

Reply:
This mirrors the language in [Flask's contributing guidelines](https://github.com/pallets/flask/blob/master/CONTRIBUTING.rst#reporting-issues). Happy to make any changes here - if you all decide to remove it, should I put in a PR to Flask to remove it there too?